src/back/spv/instructions.rs

```diff
@@ -1161,6 +1251,40 @@ mod tests {
             validate_instruction(output.as_slice(), &instruction);
         }
 
+        fn test_instruction_i_mul(&self) {
+            let instruction = super::instruction_i_mul(1, 1, 1, 1);
+            let mut output = vec![];
+
+            let requirements = SpecRequirements {
+                op: Op::IMul,
+                wc: 5,
+                type_id: true,
+                result_id: true,
+                operands: true,
+            };
+            validate_spec_requirements(requirements, &instruction);
+
+            instruction.to_words(&mut output);
+            validate_instruction(output.as_slice(), &instruction);
+        }
+
+        fn test_instruction_f_mul(&self) {
```

Comment:
similarly, having a generalized `instruction_binary_op` would allow to test all of them in a simpler way.
in general, not totally sure that all these tests are valuable, but this is up to you :)

Reply:
"Fixed" in #223